### PR TITLE
Add initial community document.

### PR DESCRIPTION
**Pull request by lseelenbinder**
This adds a simple documentation of the community structure and steering committee status.

### CODE REVIEW

**Review by nyurik — COMMENTED** [review]
All looks good to me, with the possible exception of the steering committee membership - do we want to allow more than one person from each company/organization?

File: COMMUNITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# MapLibre Community Structure
+
+MapLibre is a joint collaboration by many interested parties in GIS to ensure
+maintained, free, open source, and vendor-neutral libraries to support display
+and development of maps.
+
+No single company or individual controls more than one vote. No single entity
+has control over the development roadmap.
+
```
Comment (snickell) [inline COMMUNITY.md:9]: I like this, while it /could/ hamper a very seriously involved company at some future date, I think one-company-dominated-open-source projects are rarely healthy, and "no single company or individual" makes a lot of sense to me.

File: COMMUNITY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Steering Committee
+
+The steering committee has the rights to make decisions (via consensus or
+majority vote) regarding the direction of all MapLibre projects.
+
```
Comment (snickell) [inline COMMUNITY.md:14]: I like leaving this open ended, nice intention statement

File: COMMUNITY.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Clément Gayvallet, Jones Magloire ([Jawg](https://jawg.io))
+- Luke Seelenbinder ([Stadia Maps](https://stadiamaps.com/))
+- Marcel Normann ([WhereGroup](https://wheregroup.com/))
+- Petr Pridal, Petr Sloup ([MapTiler](https://maptiler.com/))
+- Seth Nickell ([Ceres Imaging](https://www.ceresimaging.net))
+- Yuri Astrakhan (individual contributor)
+
+## Notes
+
```
Comment (snickell) [inline COMMUNITY.md:27]: Notes both seem great and useful too

**Review by snickell — APPROVED** [review]
I think this is *awesome* @lseelenbinder , I'd love to see this merged, very clear nice writing, thanks so much for your work on this!

File: COMMUNITY.md (hunk 1 above)
Comment (lseelenbinder) [reply COMMUNITY.md:9]: Makes sense. I think this should be further clarified to state something along the lines of:

> No single company or individual controls more than one vote. No single entity has control over the development roadmap.
> 
> Thus, if one company is represented by more than one individual on the steering committee, those individuals associated with that company combine to cast a single vote. Association is defined as employment, ownership, or material interest.

I think if a single company wants to contribute significantly, their actual contributions will "vote" more than any steering committee decisions. If not, I think we've lost good faith between the members of the community, and an additional fork would be likely inevitable.

**Comment by lseelenbinder** [comment]
> All looks good to me, with the possible exception of the steering committee membership - do we want to allow more than one person from each company/organization?

Good question. I like the idea of multiple individuals, but only one vote. I suspect individual companies will have multiple interested parties, but shouldn't "control" more than a significantly involved individual contributor.

I'm open to modifying it to mean membership is very limited (one per company), but meetings are open (multiple per company are invited to attend and participate).

**Pull request merged**